server/sonar-server/src/main/java/org/sonar/server/user/index/UserResultSetIterator.java

```diff
@@ -79,15 +81,29 @@ static UserResultSetIterator create(DbClient dbClient, DbSession session, @Nulla
     }
   }
 
-  private static String createSql(@Nullable String login) {
+  private static String createSql(@Nullable List<String> logins) {
+    if (logins == null) {
+      return SQL_ALL;
+    }
+
+    List<String> sqlLogins = logins.stream()
+      .map(l -> LOGIN_FILTER)
+      .collect(Collectors.toList());
+
     String sql = SQL_ALL;
-    sql += login == null ? "" : LOGIN_FILTER;
+    sql += " WHERE ";
+    sql += "(" + OR_JOINER.join(sqlLogins) + ")";
+
     return sql;
   }
 
-  private static void setParameter(PreparedStatement stmt, @Nullable String login) throws SQLException {
-    if (login != null) {
-      stmt.setString(1, login);
+  private static void setParameters(PreparedStatement stmt, @Nullable List<String> logins) throws SQLException {
+    if (logins == null) {
+      return;
+    }
+
+    for (int i = 0; i < logins.size(); i++) {
```

Comment:
nope because `PreparedStatement.setString(…)` can throw a checked exception and you cannot have it simply in a lambda